hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/execution/traceability/SolidityAction.java

```diff
@@ -32,7 +34,7 @@ public class SolidityAction {
     private EntityId recipientContract;
     private byte[] invalidSolidityAddress;
     private final long value;
-    private long gasUsed;
+    private Long gasUsed;
```

Comment:
Why box?  Could we use a sentinel value of MIN_VALUE or -1 instead? Negative gas is _always_ invalid.

Reply:
Ok. My sense of style and readability (alluded to above) makes me want to have _all_ of these similar tests for "not set" be similar, and all but this one are nullable and tested that way.  So I wanted this one nullable too.

I don't believe it's too expensive.  One boxed value per action record.  I don't yet have a good sense of the memory pressure we're under.  Is that too much?

Reply:
huh.  ok, well replaced now with a long and IDC if it is invalid because missing => 0 turns out to be ok.